app/vendors/vendor-dashboard/profile/VendorProfileForm.tsx

```
@@ -212,104 +365,120 @@ export default function VendorProfileForm({
-                            </FormLabel>
-                          </FormItem>
-                        );
+        {/* Hidden City and State fields (populated by location selection) */}
+        <input type="hidden" {...form.register("city")} />
+        <input type="hidden" {...form.register("state")} />
+
+        {/* Service Categories with MultiSelect */}
+        <FormField
+          control={form.control}
+          name="serviceCategories"
+          render={({ field }) => (
```

ESLint

[error] 'field' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)